docs/node-mixin/config.libsonnet

```diff
@@ -60,6 +60,7 @@
     dashboardNamePrefix: 'Node Exporter / ',
     dashboardTags: ['node-exporter-mixin'],
 
-    enableLokiLogs: false,
+    // Set to true to add log panels to dashboards
+    enableLokiLogs: true,
```

Comment:
Should this be enabled by default upstream? I kept it false to help merging, since it is opinionated on loki's direction.

Reply:
well spotted, reverting to false!